spec/models/user_spec.rb

```diff
@@ -1,9 +1,11 @@
 require "rails_helper"
-
-describe User do
- context "validations" do
-
-    it { is_expected.to validate_presence_of(:username) }
-    it { is_expected.to validate_presence_of(:email) }
+RSpec.describe User do
+  subject { build(:user) }
```

Comment:
Move the subject inside the context block, and have one new line beneath it

Reply:
Fixed up commit (263fba0) in --fixup in sha: d234d15